modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/frontend/data/set/provider/CETFDSActionProvider.java

```diff
@@ -60,7 +59,10 @@ public List<DropdownItem> getDropdownItems(
 		CETFDSEntry cetFDSEntry = (CETFDSEntry)model;
 
 		if (cetFDSEntry.isReadOnly()) {
-			return Collections.emptyList();
+			return DropdownItemListBuilder.add(
+				dropdownItem -> _buildViewClientExtensionEntryAction(
+					cetFDSEntry, dropdownItem, httpServletRequest)
+			).build();
```

Comment:
This adds a "view" action also to the `actionLinkRenderer` component we use to render CE name. So, in this type of CET, clicking on the name will take us to `view` acttion, whereas in the UI-created CEs, same link will take us to the `edit` action. 
That's probably ok, nevertheless we may want to hear about it from Uge and @pablo-agulla, just in case

Reply:
Asked in slack.